# Request (empty)
---
bool success
string message
